hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpRequestLogAppender.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
I think this should be **?

Reply:
Most of the time ([example](https://gist.github.com/SparklingComet/6d4bd88ae8559eb8a66a4385f89d8250)) I see it with a single *. I know that we use some kind of mixture. In IntelliJ this is also marked as a dangling JavaDoc comment.

Reply:
But I think in HBase we usually use two stars, and also in hadoop? At least, our code template under the dev-support uses two stars.

Reply:
Ok, let me change it back to **.